vulnerability-rating-taxonomy.json

```diff
@@ -1482,7 +1486,99 @@
               "priority": 2
             }
           ]
-        }
+        },
+                 {
+          "id": "weakness_in_firmware_updates",
+          "name": "Weakness in Firmware Updates",
+          "type": "subcategory",
+          "children": [
+            {
```

Comment:
@TimmyBugcrowd. Are these changes needed ?. These are not mentioned in description of PR.